aptos-move/framework/aptos-framework/sources/configs/gas_schedule.spec.move

```diff
@@ -22,21 +22,22 @@ spec aptos_framework::gas_schedule {
         use aptos_framework::aptos_coin::AptosCoin;
         use aptos_framework::transaction_fee;
         use aptos_framework::staking_config;
-
-        pragma verify_duration_estimate = 120;
+        use aptos_framework::reconfiguration;
 
         requires exists<stake::ValidatorFees>(@aptos_framework);
         requires exists<CoinInfo<AptosCoin>>(@aptos_framework);
         include transaction_fee::RequiresCollectedFeesPerValueLeqBlockAptosSupply;
         include staking_config::StakingRewardsConfigRequirement;
 
         include system_addresses::AbortsIfNotAptosFramework{ account: aptos_framework };
+        include reconfiguration::ReconfigureEnsures;
         aborts_if len(gas_schedule_blob) == 0;
         let new_gas_schedule = util::spec_from_bytes<GasScheduleV2>(gas_schedule_blob);
         let gas_schedule = global<GasScheduleV2>(@aptos_framework);
         aborts_if exists<GasScheduleV2>(@aptos_framework) && new_gas_schedule.feature_version < gas_schedule.feature_version;
         ensures exists<GasScheduleV2>(@aptos_framework) ==> global<GasScheduleV2>(@aptos_framework) == new_gas_schedule;
         ensures exists<GasScheduleV2>(signer::address_of(aptos_framework));
+        ensures global<GasScheduleV2>(@aptos_framework) == new_gas_schedule;
```

Comment:
Do 38 && 39 => 40?

Reply:
It looks so. In that case, the following line (line #38) doesn't seem necessary.
```
        ensures exists<GasScheduleV2>(@aptos_framework) ==> global<GasScheduleV2>(@aptos_framework) == new_gas_schedule;
```